ts/features/lollipop/index.ts

```diff
@@ -107,12 +109,13 @@ export const taskRegenerateKey = (keyTag: string) =>
     TE.chain(() => TE.tryCatch(() => generate(keyTag), toCryptoError))
   );
 
-export const getLollipopHeaders = (
+export const getLollipopLoginHeaders = (
   publicKey: PublicKey,
   hashAlgorithm: string
 ) => ({
   "x-pagopa-lollipop-pub-key": Buffer.from(JSON.stringify(publicKey)).toString(
     "base64"
   ),
-  "x-pagopa-lollipop-pub-key-hash-algo": hashAlgorithm
+  "x-pagopa-lollipop-pub-key-hash-algo": hashAlgorithm,
+  "x-pagopa-app-version": isLocalEnv ? getAppVersion() : undefined
```

Comment:
I wouldn't set it to `undefined`, I'd not pass it along at all. What do you think?

Reply:
Actually if you set it undefined, it's not included in the headers (tested with proxyman)